### DISCUSSION TITLE
Нет кнопки восстановить записи при ролевом доступе на восстановления

### DISCUSSION THREAD

**Discussion by nshy** [totumonline/totum-ru-issues-and-discussions#564] [ОШБК!] [answered]
Привет!

Выставляю для таблицы "скрывать" при удалении, роли на удаление - все, роли на восстановление - некоторые. При этом нет кнопки перехода в режим восстановления.

`totum-mit 4.11.57.0`

Куда кстати лучше репортить баги, сюда или в issues репы?

**Comment by ttm-alexeypolunin** [comment]
Какой тип таблицы?

**Reply by nshy** [reply]
Простая.

**Comment by ttm-alexeypolunin** [comment]
Единственное, что можно в этом случае представить, что роль, которой дали доступ на удаление/восстановление не имеет доступа к таблице на запись (только чтение).

**Reply by nshy** [reply]
Попробовал с чистого листа - вопсроизводится. Доступ на запись есть.

- Создаю простую таблицу
- Удаление строк - скрывать
- Роли на запись - Создатель и Менеджер
- Роли на удаление - все
- Роли на восстановление - Создатель

Как Создатель не вижу кнопки переключения в режим восстановления. Стоит только убрать ролевой доступ на восстановление - кнопка появляется.

**Reply by ttm-alexeypolunin** [reply]
Ок. Так понятнее куда смотреть. Спс

**Reply by ttm-alexeypolunin** [reply]
Да, не работает, надо исправлять.

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Исправлено V5
